### PR TITLE
Increase delta between two retries for the Case session for efr32

### PR DESCRIPTION
**Pull request by mkardous-silabs**
#### Problem
* Delta between two retries for the case session is too small for constrainted device and causes case session to timeout 
* Is linked to this PR  #11435 
to help solve case timeout during resolve 
#### Change overview
* Increased delta between two retries to allow device to have time to react

#### Testing
* Manual testing with MG12 and MG24

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

File: src/messaging/ReliableMessageProtocolConfig.h [hunk 1]
```diff
@@ -54,7 +54,7 @@ namespace Messaging {
  *
  */
 #ifndef CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL
-#define CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL (300)
+#define CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL (2000)
```
Comment (jepenven-silabs) [inline src/messaging/ReliableMessageProtocolConfig.h:57]: This goes against the default definition in the spec (section 4.15.8). Please overwrite for the specific platform only.

**Review by jepenven-silabs — CHANGES_REQUESTED** [review]

File: src/messaging/ReliableMessageProtocolConfig.h (hunk 1 above)
Comment (mkardous-silabs) [reply src/messaging/ReliableMessageProtocolConfig.h]: Done

**Review by jepenven-silabs — APPROVED** [review]

**Comment by jepenven-silabs** [comment]
Fast track since it's EFR32 specific

**Comment by woody-apple** [comment]
@jepenven-silabs note: Only @andy31415 and I are approved to apply the fast track label, but I agree. Will keep it.

**Review by woody-apple — APPROVED** [review]

**Pull request merged**